### PR TITLE
add module1

### PR DESCRIPTION
**Pull request by MooDSWG**
add module1 video and edit python function name

### CODE REVIEW

File: book/module9/2 video Advance Pyhton.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Video: Advance Python features
```
Comment (attapol) [inline book/module9/2 video Advance Pyhton.md:1]: ชื่อไฟล์สะกดผิด 
ให้เปลี่ยนเป็น 2-videos-advanced-python.md

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (attapol) [inline .idea/misc.xml]: don't include this in your commit

File: .idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (attapol) [inline .idea/.gitignore]: don't include this in your commit

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (attapol) [inline .idea/modules.xml]: don't include this in your commit

File: .idea/programming-nlp-book.iml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (attapol) [inline .idea/programming-nlp-book.iml]: don't include this in your commit

File: book/comp_thinking.md [hunk 6]
```diff
@@ -12,3 +12,4 @@
 
 ```{tableofcontents}
 ```
+
```
Comment (attapol) [inline book/comp_thinking.md]: don't include this in your commit

File: book/module1/2 video comp thinking.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Video: Computational thinking
```
Comment (attapol) [inline book/module1/2 video comp thinking.md:1]: `git pull origin main` and move the content of this file to 2-videos-...

**Review by attapol — CHANGES_REQUESTED** [review]
1. be careful what you `git add`. you want to commit only the files you need. or the commit will be messy and the codebase will be messy as well. 
2. another bad thing is that you commit right onto your main branch, which makes it hard for you to go back in time. 

we will settle this later.

File: book/module1/2 video comp thinking.md (hunk 7 above)
Comment (MooDSWG) [reply book/module1/2 video comp thinking.md:1]: move to 2-videos-variables.md?

**Pull request closed without merging**